_posts/2020-03-30-20.03.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title:  "20.03: March 2020 Devblog"
+date:   2020-04-03 12:30:00 +0100
+description: |
+    "-------------------------------------------"
+image: /assets/img/posts/20.03.01/Basecode_Door_Centcom.png
+image_credit: Image credit - ----------
+post_credit: Written by - CosmicCoincidence
+---
+
+"------------------------------------------------"
+
+# March Recap
+
+Racap...
+
+## Doors
+
+**Nonanon** fixed doors...
+
+## Models
+
+New this month, **Basecode**...
+
+<div class="slideshow">
+  {% include slideshow.html %}
+  <div class="mySlides">
+    <div id="description" class="slide-description">Description 1</div>
+    <div class="slide-number">1 / 12</div>
+    {% responsive_image path: assets/img/posts/20.03.01/Basecode_Door_Centcom.png template: _includes/srcset.html %}
+  </div>
+  <div class="mySlides">
+    <div id="description" class="slide-description">Description 2</div>
+    <div class="slide-number">2 / 12</div>
+    {% responsive_image path: assets/img/posts/20.03.01/Basecode_Floors.png template: _includes/srcset.html %}
+  </div>
+  <div class="mySlides">
+    <div id="description" class="slide-description">Description 3</div>
```

Comment:
Yeah here too, be really careful with shared IDs. In HTML you should never have two elements with the same ID.

Reply:
I keep in mind that the IDs should be unique, thanks!